Documentation/ceph-dashboard.md

```diff
@@ -1,21 +1,22 @@
 ---
-title: Dashboard
+title: MGR Dashboard
```

Comment:
Can we just call it the Ceph dashboard? The fact that it's hosted by the mgr is an implementation detail.